Vagrantfile

```diff
@@ -38,11 +47,11 @@ Vagrant.configure(2) do |config|
 
   # Install Citadel
   config.vm.provision "shell", inline: <<-SHELL
-    apt-get install -y fswatch rsync jq
+    apt-get install -y fswatch rsync jq python3-dacite
```

Comment:
A recent update added python3-semver as a dependency. Can you add this here too? It doesn't crash without it, but app updates don't work properly.

`python3-jsonschema` may also be required.